### ISSUE TITLE
A11y_AzureDataStudioAAE_Sql Instance 1_Connect to Server_Usability: Focus is not visible inside the modal window when opened on pressing enter key on "connect to server" button

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15772]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yUsable;#A11yTCS;#A11ySev4;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio-AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "Sqlinstance1" and open manage and tab to "Connect to Server" and press enter
5. On pressing enter, check if the modal window have visible focus inside it.

### Actual:
Focus is not visible inside the modal window when opened on pressing enter key on "connect to server" button.

### Expected:
Focus should be visible inside the modal window when opened on pressing enter key on "connect to server" button.

### User Impact:
Keyboard user will get confused if the focus is not visible inside the modal window.

**Comment by krrish2021** [comment]
issue is not reproducing in the version 20.3.5 . Please find the attachment.

https://user-images.githubusercontent.com/85863543/127600700-badc0c69-ba14-4833-b274-6b4132f2b328.mp4
